### ISSUE TITLE
OpenSSL version is broken

### ISSUE THREAD

**Issue by notenbrood** [ThinkboxSoftware/SSLGeneration#3]
The OpenSSL version specified in requirements.txt doesn't work:
![image](https://user-images.githubusercontent.com/6082891/70316703-881d6480-181c-11ea-9a68-d8faca1249a6.png)
Steps to reproduce:

1. On Ubuntu 18.04, install pip through apt-get install python-pip
2. pip install -r requirements.txt
3. Open Python shell by typing "python" in bash
4. import OpenSSL
5. Error..

This issue also occurs in the Deadline RCS installer if you ask it generate certificates on Linux

**Comment by queglay** [comment]
Yes this is broken. The requirements file doesn't work on ubuntu 18.

instead try:
```
sudo apt-get install python-openssl
```
